cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts

```diff
@@ -202,4 +206,132 @@ describe('Test Case Expected Measure Group population values based on initial me
         cy.get(TestCasesPage.testCaseIPPCheckBox).should('not.be.checked')
 
     })
+    it('Test Case Population value options are limited to those that are only defined from Measure Group -- only required Populations on Measure Group', () => {
```

Comment:
is it possible to make this title more concise?

Reply:
I have made the newer tests a bit more concise with their names / titles.